docs: update ACL doc (#4807)
juicedata/juicefs: docs/en/security/posix_acl.md

---
title: POSIX ACLs
description: Learn about POSIX ACL support in JuiceFS and how to enable and use ACL permissions.
sidebar_position: 1
---

> POSIX ACL (Portable Operating System Interface for Unix - Access Control Lists) is an access control mechanism in Unix-like operating systems that allows for fine-grained control over file and directory access permissions.

Author: ```suggestion
POSIX ACLs (Portable Operating System Interface for Unix - Access Control Lists) are an access control mechanism in Unix-like operating systems that allows for finer-grained control over file and directory access permissions.
```
